src/Benchmarks/FortunesEfMiddleware.cs

```diff
@@ -44,9 +44,7 @@ public FortunesEfMiddleware(RequestDelegate next)
 
         private static async Task<IEnumerable<Fortune>> LoadRows(ApplicationDbContext dbContext)
         {
-            var result = new List<Fortune>();
-
-            result.AddRange(await dbContext.Fortunue.ToListAsync());
+            var result = await dbContext.Fortune.ToListAsync();
```

Comment:
I didn't do this because I wasn't sure if EF did anything to pre-size the list as part of the materialization. That isn't allowed as per the benchmark rules: "The list data structure must be a dynamic-size or equivalent and should not be dimensioned using foreknowledge of the row-count of the database table."